modules/UI/side_pannels/profile/Profile.js

```diff
@@ -54,6 +54,8 @@ export default {
     init(emitter) {
         initHTML();
 
+        const settings = getSettings(APP.store);
```

Comment:
Here, the values should come from the local user and be read only iff there is a JWT token which overrode the local user info.